tests/simulation/physics/sources/test_plane_source.py

```
@@ -0,0 +1,146 @@
+
+
+def test_uniform_plane_source_amplitude_consistency():
+    """Doubling static_amplitude_factor doubles the phasor amplitude at the detector."""
+    arrays_1x = _build_and_run(amplitude_factor=1.0)
+    arrays_2x = _build_and_run(amplitude_factor=2.0)
+
+    # Extract Ex phasor (may have trailing dims even with reduce_volume)
+    p1 = complex(arrays_1x.detector_states["phasor"]["phasor"][0, 0].ravel()[0])
+    p2 = complex(arrays_2x.detector_states["phasor"]["phasor"][0, 0].ravel()[0])
+
+    assert abs(p1) > 1e-30, "1× phasor amplitude is zero"
```

Ruff (0.15.9)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)